bindings/wasm/examples/src/1_advanced/7_status_list_2021.ts

```diff
@@ -0,0 +1,153 @@
+// Copyright 2020-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    Credential,
+    EdDSAJwsVerifier,
+    FailFast,
+    JwkMemStore,
+    JwsSignatureOptions,
+    JwtCredentialValidationOptions,
+    JwtCredentialValidator,
+    KeyIdMemStore,
+    StatusCheck,
+    StatusList2021,
+    StatusList2021Credential,
+    StatusList2021CredentialBuilder,
+    StatusList2021Entry,
+    StatusPurpose,
+    Storage,
+} from "@iota/identity-wasm/node";
+import { Client, MnemonicSecretManager, Utils } from "@iota/sdk-wasm/node";
+import { API_ENDPOINT, createDid } from "../util";
+
+export async function statusList2021() {
```

Comment:
All the comments from previous reviews on the rust example should also be reflected on the Wasm example. We try to keep the rust and TS example as similar as we can.